### PR TITLE
Added a new sound to the nuke explosion

### PR DESCRIPTION
**Pull request by Jujumatic**
I've added a new secret sound that replaces the alarm right before the nuke explodes - current testing has determined that I managed to nail the timing on the explosion PERFECTLY with the sound I made

### CODE REVIEW

**Comment by praisenarsie** [comment]
poopoo joke

**Comment by Jujumatic** [comment]
poopoo opinion lol

**Comment by Pyko1** [comment]
not really big fan of the sound.

**Comment by Jujumatic** [comment]
it's an 8% chance to happen so don't expect it to suddenly be popping up much. i added this as a joke but i can understand if it's not funny

File: code/datums/cinematic.dm [hunk 1]
```diff
@@ -104,7 +104,9 @@ GLOBAL_LIST_EMPTY(cinematics)
 	flick("intro_nuke",screen)
 	sleep(35)
 	flick("station_explode_fade_red",screen)
-	cinematic_sound(sound('sound/effects/explosion_distant.ogg'))
+	if (sound_to_playing_players('sound/effects/explosion_secret.ogg'))
```
Comment (Carbonhell) [inline code/datums/cinematic.dm:107]: what? is something missing here?

**Comment by Carbonhell** [comment]
I'm not sure about this either honestly, -1
closing since it has two -1s

**Pull request closed without merging**